packages/components/app/styles/components/alert.scss

```diff
@@ -58,3 +57,111 @@
     }
   }
 }
+
+//
+// COLOR & TYPE
+//
+.hds-alert--color-critical {
+  background-color: var(--token-color-surface-critical);
+
+  &.hds-alert--type-page {
+    box-shadow: 0px 1px 0px 0px var(--token-color-border-critical);
+  }
+
+  &.hds-alert--type-inline { 
+    border-color: var(--token-color-border-critical);
+  }
+
+  .hds-alert__icon, .hds-alert__title {
+    color: var(--token-color-foreground-critical-on-surface);
+  } 
+}
+
+.hds-alert--color-warning {
+  background-color: var(--token-color-surface-warning);
+
+  &.hds-alert--type-page {
+    box-shadow: 0px 1px 0px 0px var(--token-color-border-warning);
+  }
+
+  &.hds-alert--type-inline { 
+    border-color: var(--token-color-border-warning);
+  }
+
+  .hds-alert__icon, .hds-alert__title {
+    color: var(--token-color-foreground-warning-on-surface);
+  }
+}
+
+// TODO: Check all neutral tokens, cause don't follow pattern of other `color`s 
+.hds-alert--color-neutral {
+  background-color: var(--token-color-surface-faint);
+
+  &.hds-alert--type-page {
+    // TODO: Check if should be a token for 0px 1px 0px 0px thruout, and rgba
+    box-shadow: 0px 1px 0px 0px rgba(59, 61, 69, 0.4);
```

Comment:
re: This TODO

I wasn't sure if I was missing something, should the box-shadow px and this rgba color be a design token? 

(If not, should I convert to rem, our scale is 1rem = 16px? There could be a few other non-rem's in there as well) @didoo @cveigt

Reply:
0350cb1dce06bc08923453f180715d4b35f34775